docs/en_us/platform_api/source/shared/courses_response_values.rst

```diff
@@ -0,0 +1,223 @@
+.. _Course:
+
+Course
+*******
+
+* count (integer): The number of courses in the catalog.
+* next (string): The URL for the next page of results.
+* previous (string): The URL for the previous page of results.
+* results (array): See :ref:`Results`.
+
+.. _Results:
+
+Results
+*******
+
+Parent: ``course``
+
+A list of courses in the catalog.
+
+* key (string): The unique identifier for the course.
+* title (string): The title of the course.
+* short_description (string): The short description of the course and its
+  content.
+* full_description (string): The long description of the course and its
+  content.
+* level_type (ENUM string): ---DESCRIPTION---
```

Comment:
This is a level of difficulty or rigor as estimated by the course team.  We will only allow a set list of accepted values here.  I believe they are:
`high_school`
`introductory`
`intermediate`
`advanced`